packages/commonwealth/client/scripts/state/api/threads/getThreadsById.ts

```diff
@@ -0,0 +1,45 @@
+import { useQuery } from '@tanstack/react-query';
+import axios from 'axios';
+import Thread from 'models/Thread';
+import app from 'state';
+import { ApiEndpoints } from 'state/api/config';
+
+const THREAD_STALE_TIME = 180000; // 3 minutes
```

Comment:
I really think we should use no longer than 5-10 seconds as the stale time / caching time for this PR. This applies to all cache TTLs as configured in this PR.

The reason is, as I described in another comment, that we want new threads to appear basically immediately when someone else creates or updates them.

The long-term solution will be to move to a fully event-driven pattern, where we have a websocket connection that pushes actions that invalidate the cache on receipt, but for the time being we should err somewhat on the side of responsiveness over data saving.